src/os/open.zig

```diff
@@ -12,7 +12,7 @@ pub fn open(alloc: Allocator, url: []const u8) !void {
     // the process to exit to collect stderr.
     const argv, const wait = switch (builtin.os.tag) {
         .linux => .{ &.{ "xdg-open", url }, false },
-        .macos => .{ &.{ "open", url }, true },
+        .macos => .{ &.{ "open -t", url }, true },
```

Comment:
Thanks for working on this! While using `open -t` directly in the implementation works, it might be better to keep the `open` function generic and pass the `-t` flag as a parameter instead. This way:

1. The `open` function remains flexible for different use cases
2. Other callers of `open` won't be forced to use a text editor
3. The specific opening behavior stays controlled by the caller